py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -7,12 +7,12 @@
 import sys
 
 from typing import NamedTuple
-from lambda_types import LambdaDict, LambdaContext
 
 from py_gtfs_rt_ingestion import ArgumentException
 from py_gtfs_rt_ingestion import DEFAULT_S3_PREFIX
 from py_gtfs_rt_ingestion import batch_files
 from py_gtfs_rt_ingestion import file_list_from_s3
+from py_gtfs_rt_ingestion.lambda_types import LambdaDict, LambdaContext
```

Comment:
This import pattern is different from the other items being brought in from the library, but I'm fine with it if you are.